sunpy/visualization/animator/base.py

```diff
@@ -292,15 +298,15 @@ def _add_widgets(self):
         self.sliders = []
         self.slider_buttons = []
         for i in range(self.num_sliders):
-            x = i * 2
+            y = i * 2
             self.sliders.append(self.fig.add_axes((0., 0., 0.01+i/10., 1.)))
             if self.num_buttons == 0:
-                nx1 = 1
+                nx1 = 3
             else:
-                nx1 = -3
-            locator = self.divider.new_locator(nx=0, ny=x, nx1=nx1)
+                nx1 = -2
+            locator = self.divider.new_locator(nx=2, ny=y, nx1=nx1)
             self.sliders[-1].set_axes_locator(locator)
-            sframe = widgets.Slider(self.sliders[-1], f"{i:d}",
+            sframe = widgets.Slider(self.sliders[-1], "",
```

Comment:
What's going on in these lines and the ~10 above?

Reply:
It is computing the layout for the sliders. I am not going to attempt to explain it in detail in a GH comment, but basically the whole thing works by generating a  x by y grid and then placing axes on the grid which span a given number of rows and columns. So calls to `new_locator` specify the start index for the axes and how many rows and columns it spans. (`nx1`, `ny1`).